civiclens/collect/move_data_from_api_to_database.py

```diff
@@ -209,6 +210,52 @@ def get_most_recent_doc_comment_date(doc_id: str) -> str:
     return most_recent_date
 
 
+def clean_docket_data(docket_data: json) -> None:
```

Comment:
This function looks as if it does nothing.

Reply:
yes, this placeholder method alludes to my goal of data classes with static methods